Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -13,7 +13,28 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
 
         [SerializeField]
         protected GameObject visuals;
-        
+
+        /// <summary>
+        /// Maximum distance a which a touchable surface can be interacted with.
```

Comment:
What's the difference?

Reply:
You could have for example a touchable volume instead of a touchable surface.

Reply:
It's based on the virtual distance function of the touchable, to the PokePointer it makes no difference how its implemented. The explanation belongs into the implementation class to explain how that distance is computed.

Semantic nitpick: i would call it "planar surface" vs. "collider surface" instead of "volume".